From 94b4451f4c12cb3ca0fd84847446ec2aa413cdb6 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:06:13 -0400
Subject: [usb] uas: uas_alloc_cmd_urb - drop unused stream_id parameter

Message-id: <1410887463-20674-302-git-send-email-dzickus@redhat.com>
Patchwork-id: 95580
O-Subject: [RHEL7 PATCH 301/591] uas: uas_alloc_cmd_urb: drop unused stream_id parameter
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit a887cd366347c38607f0d9c28ca2baed40cac8fc
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Thu Oct 17 19:47:28 2013 +0200

    uas: uas_alloc_cmd_urb: drop unused stream_id parameter

    The cmd endpoint never has streams, so the stream_id parameter is unused.

    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/storage/uas.c b/drivers/usb/storage/uas.c
index dcaf611..5eacb80 100644
--- a/drivers/usb/storage/uas.c
+++ b/drivers/usb/storage/uas.c
@@ -454,7 +454,7 @@ static struct urb *uas_alloc_sense_urb(struct uas_dev_info *devinfo, gfp_t gfp,
 }
 
 static struct urb *uas_alloc_cmd_urb(struct uas_dev_info *devinfo, gfp_t gfp,
-     struct scsi_cmnd *cmnd, u16 stream_id)
+     struct scsi_cmnd *cmnd)
 {
  struct usb_device *udev = devinfo->udev;
  struct scsi_device *sdev = cmnd->device;
@@ -626,8 +626,7 @@ static int uas_submit_urbs(struct scsi_cmnd *cmnd,
  }
 
  if (cmdinfo->state & ALLOC_CMD_URB) {
-  cmdinfo->cmd_urb = uas_alloc_cmd_urb(devinfo, gfp, cmnd,
-           cmdinfo->stream);
+  cmdinfo->cmd_urb = uas_alloc_cmd_urb(devinfo, gfp, cmnd);
   if (!cmdinfo->cmd_urb)
    return SCSI_MLQUEUE_DEVICE_BUSY;
   cmdinfo->state &= ~ALLOC_CMD_URB;
-- 
1.7.1